### PR TITLE
Remove condition tfm for single tfm

### PR DESCRIPTION
**Pull request by livarcocc**
Fixes https://github.com/dotnet/cli/issues/5143 and https://github.com/dotnet/cli/issues/5140

@piotroko @piotrpMSFT @jonsequitur @jgoshi @krwq

### CODE REVIEW

File: src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs [hunk 1]
```diff
@@ -59,8 +59,11 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
             foreach (var targetFramework in targetFrameworks)
             {
                 MigrationTrace.Instance.WriteLine(String.Format(LocalizableStrings.MigratingFramework, targetFramework.FrameworkName.GetShortFolderName()));
-                
-                MigrateImports(migrationRuleInputs.CommonPropertyGroup, targetFramework);
+
+                MigrateImports(
+                    migrationRuleInputs.CommonPropertyGroup,
+                    targetFramework,
+                    migrationRuleInputs.ProjectContexts.Count() > 1);
```
Comment (piotrpMSFT) [inline src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs]: Can we make this boolean calculation an extension method on MigrationRuleInputs? Or a `get`-only property?

File: src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs [hunk 2]
```diff
@@ -161,13 +164,14 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
 
         private void MigrateImports(
             ProjectPropertyGroupElement commonPropertyGroup,
-            TargetFrameworkInformation targetFramework)
+            TargetFrameworkInformation targetFramework,
+            bool isMultiTFM)
```
Comment (piotrpMSFT) [inline src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs]: bool conditionOnFramework?

File: test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigratePackageDependencies.cs [hunk 3]
```diff
@@ -201,7 +201,27 @@ public void It_migrates_imports_per_framework()
         }
 
         [Fact]
-        public void It_auto_add_desktop_references_during_migrate()
+        public void ItDoesNotConditionToPackageTargetFallBackWhenMigratingASingleTFM()
```
Comment (piotrpMSFT) [inline test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigratePackageDependencies.cs]: It does not `add` conditi... ?

File: test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs [hunk 4]
```diff
@@ -15,6 +15,8 @@ namespace Microsoft.DotNet.ProjectJsonMigration.Tests
 {
     public class GivenThatIWantToMigrateScripts : TestBase
     {
+        private const bool IsMultiTFM = true;
```
Comment (piotrpMSFT) [inline test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs]: Seems we should have a default value. Having to pass this in every test distracts from what is being tested.

File: src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs (hunk 1 above)
Comment (livarcocc) [inline src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs]: For sure, I actually had a mental note to do that and forgot. Thanks for pointing it out.

**Comment by livarcocc** [comment]
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build

**Comment by livarcocc** [comment]
@srivatsn

**Pull request merged**